samigo/samigo-app/src/webapp/WEB-INF/faces-config.xml

```diff
@@ -1005,6 +1005,15 @@
       </converter-class>
    </converter>
 
+   <converter>
+     <converter-id>
+       org.sakaiproject.tool.assessment.jsf.converter.UsermameConverter
```

Comment:
```suggestion
       org.sakaiproject.tool.assessment.jsf.converter.UsernameConverter
```

Reply:
Nah, I'll remove this.

Reply:
Voilá https://github.com/sakaiproject/sakai/pull/12213/commits/f2e4a110a3c2416aa8be4dc729d274d5d875fa3d!